pkg/modelregistry/modelregistry_test.go

```diff
@@ -134,6 +135,614 @@ func Test_Schema(t *testing.T) {
 	}
 }
 
+func Test_State(t *testing.T) {
```

Comment:
It would be good on at least some of them to check that the valuetype returned was correct